umake/frameworks/ide.py

```diff
@@ -749,9 +750,11 @@ def check_external_license(self, result):
 
     def post_install(self):
         """Create the Visual Studio Code launcher"""
+        exec_path = os.path.join(self.install_path, "Code")
         create_launcher(self.desktop_filename, get_application_desktop_file(name=_("Visual Studio Code"),
                         icon_path=os.path.join(self.install_path, "resources", "app", "resources", "linux",
                                                "vscode.png"),
-                        exec=os.path.join(self.install_path, "Code"),
+                        exec=exec_path,
                         comment=_("Visual Studio focused on modern web and cloud"),
                         categories="Development;IDE;"))
+        add_exec_link(exec_path, self.desktop_filename.split('.')[0])
```

Comment:
I would rather add the add_exec_link call to BaseInstaller, on condition there is a desktop file, and matching its exec path (that can be a new flag in __init__(). What do you think?